tests/unit/document/test_converters.py

```diff
@@ -274,3 +274,17 @@ def test_glb_converters(uri, chunk_num):
     doc.load_uri_to_point_cloud_tensor(2000, as_chunks=True)
     assert len(doc.chunks) == chunk_num
     assert doc.chunks[0].tensor.shape == (2000, 3)
+
+
+@pytest.mark.parametrize(
+    'uri',
+    [
+        os.path.join(cur_dir, 'toydata/test.glb'),
+        'https://github.com/jina-ai/docarray/raw/main/tests/unit/document/toydata/test.glb',
```

Comment:
do not enter such dependency, there is no need and can make the test flaky if github is down or something